packages/gui/src/components/nfts/detail/NFTDetailV2.tsx

```diff
@@ -75,13 +165,15 @@ export default function NFTDetail() {
             position="relative"
           >
             {nft && (
-              <Box onClick={handleShowFullScreen} sx={{ cursor: 'pointer' }}>
```

Comment:
Image NFTs should support onClick to show a larger version of the image.

Reply:
Applied your patch.